docs/test_scripts/test_basic_inference.py

```diff
@@ -0,0 +1,43 @@
+# [imports]
+import torch
+import numpy as np
+import modulus
+import matplotlib.pyplot as plt
+from modulus.datapipes.benchmarks.darcy import Darcy2D
+from modulus.models.fno.fno import FNO
+
+# [imports]
+
+# [code]
+
+model = FNO(
+    in_channels=1,
+    out_channels=1,
+    decoder_layers=1,
+    decoder_layer_size=32,
+    dimension=2,
+    latent_channels=32,
+    num_fno_layers=4,
+    num_fno_modes=12,
+    padding=5,
+).to("cuda")
+
+# Save the checkpoint. For demo, we will just save untrained checkpoint, 
+# but in typical workflows is saved after model training. 
+model.save("untrained_checkpoint.mdlus")
+
+# Inference code
+
+# The parameters to instantitate the model will be loaded from the checkpoint
+model_inf = FNO.from_checkpoint("untrained_checkpoint.mdlus").to("cuda")
```

Comment:
Can you change this to `Module.from_checkpoint` so you don't need to know the exact class needed at inference time? Just need the .mdlus checkpoint and that's it.

Reply:
Good catch, done